jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/impl/util/HumanTaskHandlerHelper.java

```diff
@@ -104,33 +105,51 @@ public static Deadlines setDeadlines(Map<String, Object> parameters, List<Organi
 	            Deadline taskDeadline = null;
 	            
 	            for (String expiresAt : expireElements) {
-	            	logger.debug("Expires at is {}", expiresAt);
-	                taskDeadline = TaskModelProvider.getFactory().newDeadline();
-	                if (businessCalendar != null) {
-	                	taskDeadline.setDate(businessCalendar.calculateBusinessTimeAsDate(expiresAt));
-	                } else {
-	                	taskDeadline.setDate(new Date(System.currentTimeMillis() + TimeUtils.parseTimeString(expiresAt)));
-	                }
-	                logger.debug("Calculated date of execution is {} and current date {}", taskDeadline.getDate(), new Date());
-	                List<Escalation> escalations = new ArrayList<Escalation>();
-	                
-	                Escalation escalation = TaskModelProvider.getFactory().newEscalation();
-	                escalations.add(escalation);
-	                
-	                escalation.setName("Default escalation");
-	                
-	                taskDeadline.setEscalations(escalations);
-	                escalation.setReassignments(parseReassignment(actionComponent));
-	                escalation.setNotifications(parseNotifications(actionComponent, businessAdministrators));
-	                
-	                deadlines.add(taskDeadline);
+	            	if(businessCalendar != null) {
+						deadlines.add(getNewDeadline(expiresAt, businessCalendar.calculateBusinessTimeAsDate(expiresAt.trim()), actionComponent, businessAdministrators));
+					} else {
+						if (DateTimeUtils.isRepeatable(expiresAt.trim())) {
+							int repeatLimit = getDeadlineRepeatLimit(expiresAt.trim());
+							if(repeatLimit > 0) {
+								Date tmpDate = null;
```

Comment:
I guess you need to set the tmpDate based on the optional start date from repeatable fom iso format